Fortune.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+
+"""Fortune wrapper extension"""
+
+from albertv0 import *
+from shutil import which
+import subprocess as sp
+
+__iid__ = "PythonInterface/v0.1"
+__prettyname__ = "Fortune"
+__version__ = "1.0"
+__trigger__ = "fortune"
+__author__ = "Kelvin Wong"
+__dependencies__ = ["fortune"]
+
+if which("fortune") is None:
+    raise("'fortune' is not in $PATH.")
```

Comment:
Tried the extension **without** fortune being installed, got this error:

```bash
12:10:56 [WARN] [Fortune.py] TypeError: exceptions must derive from BaseException

At:
  ./Fortune.py(17): <module>
  <frozen importlib._bootstrap>(219): _call_with_frames_removed
  <frozen importlib._bootstrap_external>(678): exec_module
```

So I think `raise` is not enough here for proper error handling.